### DISCUSSION TITLE
Graceful stop in web browser?

### DISCUSSION THREAD

**Discussion by vojty** [emilk/egui#242] [Q&A]
Hello, is there a way how to stop the egui in the browser?
I'm getting when I try to leave the page.
```
panicked at 'called `Option::unwrap()` on a `None` value', /Users/xxx/.cargo/registry/src/github.com-1ecc6299db9ec823/egui_web-0.10.0/src/lib.rs:118:44
```

Something like:
```
let runner_ref = egui_web::start(canvas_id, Box::new(app)).unwrap();
// ...
egui_web::stop(runner_ref);
```

**Comment by emilk** [comment]
No, I haven't added this feature yet. All the events listeners in https://github.com/emilk/egui/blob/master/egui_web/src/lib.rs need to be removed somehow.
